shopfloor/models/stock_move.py

```diff
@@ -36,3 +36,36 @@ def _action_done(self, cancel_backorder=False):
                 if picking.state == "done":
                     picking.action_done()
         return moves
+
+    def action_done_in_picking(self):
+        """Put the moves in a separate transfer and validate them."""
+        moves = self.filtered(lambda m: m.state == "assigned")
```

Comment:
Should I add `partially_available` too?

Reply:
I guess it depends of the answer to https://github.com/camptocamp/wms/pull/72#discussion_r489475510 :)

Reply:
I stick for `assigned` only then